tests/contrib/boto/test.py

```diff
@@ -70,6 +72,7 @@ def test_s3_client(self):
         s3.get_all_buckets()
         spans = writer.pop()
         assert spans
+        eq_(len(spans), 1)
```

Comment:
Not related to this line (but I can't add a comment to the line :79 ... I think we should remove that comment or make it work properly?

For reference I'm talking about:
```python
# eq_(span.get_tag('host'), 's3.amazonaws.com'). not same answers PY27, PY34..
```